### PR TITLE
Add custom vision classifier and update XF sample to use it

### PR DESCRIPTION
**Pull request by MaikuB**

### CODE REVIEW

File: src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj [hunk 1]
```diff
@@ -13,6 +13,10 @@
     <Compile Include="Platforms\Uap\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard'))">
+    <Compile Include="Platforms\NetStandard\**\*.cs" />
```
Comment (nickrandolph) [inline src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj]: Why is this being included here and then the file being removed below?

File: src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj [hunk 2]
```diff
@@ -51,4 +55,12 @@
     <Compile Include="Platforms\Xamarin-common\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Platforms\NetStandard\CrossCustomVisionClassifier.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="Interfaces\ICustomVisionClassifier.cs" />
```
Comment (nickrandolph) [inline src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj]: Should this still be here?

File: src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj (hunk 1 above)
Comment (MaikuB) [reply src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj]: hmm think VS is doing some weird stuff, will take a look

Comment (nickrandolph) [reply src/BuildIt.ML/BuildIt.ML/BuildIt.ML.csproj]: We should probably add this to most of the BuildIt csproj files

File: src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs [hunk 3]
```diff
@@ -2,21 +2,26 @@
...
 using Android.OS;
 using BuildIt.ML.Sample.UI;
+using Plugin.CurrentActivity;
+using Plugin.Permissions;
 
 namespace BuildIt.ML.Sample.Droid
 {
     [Activity(Label = "BuildIt.ML.Sample", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
-    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsApplicationActivity
```
Comment (nickrandolph) [inline src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs]: why this change? we should be using AppCompat where possible

File: src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs [hunk 4]
```diff
@@ -2,21 +2,26 @@
...
     {
         protected override void OnCreate(Bundle bundle)
         {
-            TabLayoutResource = Resource.Layout.Tabbar;
-            ToolbarResource = Resource.Layout.Toolbar;
-
             base.OnCreate(bundle);
 
             global::Xamarin.Forms.Forms.Init(this, bundle);
+            CrossCurrentActivity.Current.Init(this, bundle);
```
Comment (nickrandolph) [inline src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs]: Is there no way that we can get around having this call here?

File: src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs (hunk 3 above)
Comment (MaikuB) [reply src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs]: app was crashing originally, will see if it can work again upon reverting that

File: src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs (hunk 4 above)
Comment (MaikuB) [reply src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs]: to be clear, the camera plugin depends on the permissions plugin, which in turns depends on that activity one. so no, it won't go away in the sample if we keep using the permissions plugin (which we do for BuildIt.Forms)
